README.md

```diff
@@ -87,14 +87,22 @@ Please ensure you have only one pull request per feature.
 
 ## Testing
 
-Please note that testing requires launching the complete DSP API stack which is based on docker images. 
-Therefore, we recommend installing the [docker desktop client](https://www.docker.com/products).  
-To run the complete test suite:
+The tests of this repository 
```

Comment:
Here I use Semantic Line Breaks (https://sembr.org/) for the first time. It sounds convincing, but I'm curious to hear your feedback.

One of the big advantages, which is not even described in the specification: we will get cleaner git diffs, because changing a single word in the middle of a paragraph won't force us any more to change all line breaks of that paragraph.